src/app/core/store/data/data.state.ts

```diff
@@ -91,6 +91,10 @@ export class DataState extends NgxsDataRepository<DataStateModel> {
   private readonly _listDataQueryStatus$ = new ReplaySubject<DataQueryState>(1);
   /** Implementation subject for aggregateDataQueryStatus$. */
   private readonly _aggregateDataQueryStatus$ = new ReplaySubject<DataQueryState>(1);
+  /** Implementation subject for termOccurencesDataQueryStatus$. */
+  private readonly _termOccurencesDataQueryStatus$ = new ReplaySubject<DataQueryState>(1);
+  /** Implementation subject for ontologyTermsDataQueryStatus$. */
+  readonly _ontologyTermsFullData$: Observable<Record<string, number>>;
```

Comment:
Remove preceding `_`. This is not private nor is there a name clash. Also the comment/doc is incorrect